packages/waku/src/lib/builder/build.ts

```diff
@@ -310,6 +313,7 @@ const buildSsrBundle = async (
     },
     publicDir: false,
     build: {
+      emptyOutDir: !partial,
```

Comment:
What I'm not sure is if `partial` is the right word for this.
Isn't this mean we keep garbage files?
Anyway, that's why I preferred "experimental partial".

Reply:
> Isn't this mean we keep garbage files?

We do. It wont delete anything, and if a page disappears from the router definition in a subsequent build, the static build artifacts will remain. Thats why its not `update` or `incremental`.
I _thought_ that i've seen this name for this behavior, but I can't remember where. `rsync` calls it `--ignore-existing`. 🤔

Reply:
Now I remember 😄 
I got it from the Drupal config import command: https://www.drush.org/12.x/commands/config_import/#options

But the behavior is not even the same, because in this case it does add and update, not delete 🤦‍♂️ 

> There are only two hard things in Computer Science: cache invalidation and naming things.

This PR has both!

Reply:
> This PR has both!

lol 🤣